### PR TITLE
test(cluster): add testcases

### PR DESCRIPTION
**Pull request by z8674558**
### TODO

add more testcases 
~~cluster connection with TLS~~ done
~~cover https://github.com/emqx/emqx-auth-redis/pull/187~~ done
sentinel

### CODE REVIEW

File: test/emqx_auth_redis_SUITE.erl [hunk 1]
```diff
@@ -69,21 +69,38 @@ set_special_configs(_App) ->
...
+    % {ok, Connection} = ?POOL(?APP),
     %% Users
-    [eredis:q(Connection, ["HMSET", Key|FiledValue]) || {Key, FiledValue} <- ?INIT_AUTH],
-
-    %% ACLs
-    emqx_modules:load_module(emqx_mod_acl_internal, false),
-    Result = [eredis:q(Connection, ["HSET", Key, Filed, Value]) || {Key, Filed, Value} <- ?INIT_ACL],
+    % [eredis:q(Connection, ["HMSET", Key|FiledValue]) || {Key, FiledValue} <- ?INIT_AUTH],
+    {ok, Server} = application:get_env(?APP, server),
+    Result = case proplists:get_value(type, Server) of
```
Comment (HJianBo) [inline test/emqx_auth_redis_SUITE.erl:76]: Could you encapsulate these three case statements into a common method? 
It's a little more straightforward

**Review by HJianBo — CHANGES_REQUESTED** [review]

File: test/emqx_auth_redis_SUITE.erl (hunk 1 above)
Comment (z8674558) [reply test/emqx_auth_redis_SUITE.erl]: Sure that is a nice idea. Now I've added a method `q/1` which only takes a command as an argument.

Comment (HJianBo) [reply test/emqx_auth_redis_SUITE.erl]: Cool

**Review by HJianBo — APPROVED** [review]

**Comment by HJianBo** [comment]
Great job, so our use cases cover most of the scenarios!

**Pull request merged**